cfme/containers/provider/__init__.py

```diff
@@ -187,17 +201,26 @@ def __init__(
         self.zone = zone
         self.endpoints = endpoints
         self.provider_data = provider_data
-        self.metrics_type = (metrics_type if self.appliance.version >= '5.9' else None)
+        self.metrics_type = metrics_type
+        self.alerts_type = alerts_type
 
     @property
     def view_value_mapping(self):
-        return {
+        mapping = {
             'name': self.name,
             'prov_type': self.type,
-            'zone': self.zone,
-            'metrics_type': self.metrics_type
+            'zone': self.zone
         }
 
+        if self.appliance.version >= '5.9':
+            mapping['metrics_type'] = self.metrics_type
+            mapping['alerts_type'] = self.alerts_type
+        else:
```

Comment:
This is not too important, but why not to initialize the fiels to `None`, so no `else` would be needed

```
        mapping = {
             'name': self.name,
            ...
            'metrics_type': None
            'alerts_type': None
            ...
            'zone': self.zone
         }
```

Reply:
I removed this from  __init__.  Metrics and Alters should be in the OpenShiftProvider init